src/sentry/static/sentry/app/views/projectFilters/projectFilters.jsx

```diff
@@ -0,0 +1,404 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import moment from 'moment';
+
+import ApiMixin from '../../mixins/apiMixin';
+import IndicatorStore from '../../stores/indicatorStore';
+import GroupTombstones from '../../components/groupTombstones';
+import LoadingError from '../../components/loadingError';
+import LoadingIndicator from '../../components/loadingIndicator';
+import ProjectState from '../../mixins/projectState';
+import StackedBarChart from '../../components/stackedBarChart';
+import {t} from '../../locale';
+import {intcomma} from '../../utils';
+
+import FilterRow from './filterRow';
+import LegacyBrowserFilterRow from './legacyBrowserFilterRow';
+import ProjectFiltersSettingsForm from './projectFiltersSettingsForm';
+
+const ProjectFilters = React.createClass({
+  mixins: [ApiMixin, ProjectState],
+
+  getInitialState() {
+    let until = Math.floor(new Date().getTime() / 1000);
+    let since = until - 3600 * 24 * 30;
+
+    return {
+      expected: 3,
+      loading: true,
+      error: false,
+      statsError: false,
+      filterList: [],
+      querySince: since,
+      queryUntil: until,
+      rawStatsData: null,
+      formattedData: [],
+      projectOptions: {},
+      blankStats: true,
+      activeSection: 'data-filters',
+      tombstones: [],
+      tombstoneError: false
+    };
+  },
+
+  componentWillMount() {
+    this.fetchData();
+  },
+
+  componentDidUpdate(prevProps) {
+    if (!this.state.loading && !this.state.formattedData) {
+      this.render();
+    }
+  },
+
+  getStatOpts() {
+    return {
+      'ip-address': 'IP Address',
+      'release-version': 'Release',
+      'error-message': 'Error Message',
+      'browser-extensions': 'Browser Extension',
+      'legacy-browsers': 'Legacy Browser',
+      localhost: 'Localhost',
+      'web-crawlers': 'Web Crawler',
+      'invalid-csp': 'Invalid CSP',
+      cors: 'CORS',
+      blacklisted: 'Filtered Events' //TODO(maxbittker) this is only needed until October 10th, 2017
+    };
+  },
+
+  formatData(rawData) {
+    let cutOverDate = moment([2017, 8, 11]); // date when detailed stats started being recorded
+
+    return Object.keys(this.getStatOpts()).map(stat => {
+      return {
+        data: rawData[stat].map(([x, y]) => {
+          if (y > 0) {
+            this.setState({blankStats: false});
+          }
+
+          //TODO(maxbittker) this is only needed until October 10th, 2017 :
```

Comment:
reminder to add a reminder for this or else this will go forgotten, could even create a PR for it now

Reply:
i'll add a calendar reminder